test/functional/hooks.js

```diff
@@ -160,5 +162,115 @@ describe('library hooks', () => {
       });
     });
   });
-});
 
+  describe('for being notified when nodes are created and added to DOM', () => {
+    var created;
+    beforeEach(() => {
+      notifications.nodesCreated = function(nodes) {
```

Comment:
should remove this hook in an afterEach so that the tests are completely independant